rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -962,6 +962,22 @@ public void updateCollections(Transaction rskTx) throws IOException {
         updateFederationCreationBlockHeights();
     }
 
+    protected void createAndProcessSvpFundTransactionWithoutSignatures() throws InsufficientMoneyException, IOException {
```

Comment:
Where is this used?

Same comment regarding the `and` in the method name, although in this case I think it'd be wiser to renamed rather than split.

Let's discuss this a bit, I guess it depends a bit on who the caller is going to be

Reply:
Should we emit release_requested event also? Where does that come into play? And `pegout_transaction_created` event

Reply:
the `protected` is for testing the method.
`release_requested` event can't be emitted because the rskTxHash is null right?
`pegout_transaction_created` should be added. Thanks!

Reply:
regarding the method name / behavior, yep maybe we can discuss it a bit :)